public/locales/en/translation.json

```diff
@@ -187,26 +187,22 @@
   },
   "retrocert-login": {
     "title": "Retroactive Certification for Unemployment",
-    "subheader": "Help us find you",
-    "help-text": "We need to make sure your identity matches EDD records so that we show you accurate records.",
-    "last-name-label": "Last name",
-    "dob-heading": "Date of Birth",
+    "help": "You are required to certify for Unemployment Insurance (UI) and Pandemic Unemployment Assistance (PUA) benefits from the week ending February 9, 2020, through week ending May 9, 2020.",
+    "instructions": "Enter your information to confirm if you have any benefit weeks that you need to retroactively certify.",
+    "last-name-label": "* Last Name",
+    "dob-heading": "* Date of Birth",
```

Comment:
Oh yeah, can we take the * out of the translation and move it to the .js file? Then we can put <span aria-label="required field" title="required field">*</span> to provide accessible text (https://github.com/cagov/unemployment/issues/319).